csrc/multidevice/communication.cpp

```diff
@@ -325,8 +325,8 @@ c10::intrusive_ptr<c10d::Work> postAllgather(
     c10d::Backend* backend,
     at::Tensor input_tensor,
     at::Tensor output_tensor) {
-  auto splits = at::split(output_tensor, /*split_size=*/1, /*dim=*/0);
-  assertBufferCount(splits, communication->team().size());
+  auto splits =
```

Comment:
what is the motivation for this change?

Reply:
split takes the size of each split and tensor_split takes the number of splits. Previously, it was easier to use split because the split size must be 1. With this PR, it's easier to use tensor_split because the split size is not necessarily 1 but the number of splits must be the team size.